providers/base/units/dock/jobs.pxu

```diff
@@ -2010,7 +2010,6 @@ plugin: user-interact
 id: dock/thunderbolt3-remove
 category_id: dock
 estimated_duration: 20.0
-depends: dock/thunderbolt3-insert
```

Comment:
It looks like I accidentally remove the depends. Let's put it back!
Anyway, this job is not included in my test plan `dock-cert-full`, so it will not affect anything now.